pkg/brokerapi/service_binding.go

```diff
@@ -40,7 +40,6 @@ type BindingRequest struct {
 // CreateServiceBindingResponse represents a response to a service binding
 // request
 type CreateServiceBindingResponse struct {
-	// SyslogDrainURL string      `json:"syslog_drain_url, omitempty"`
```

Comment:
I think this was there on purpose as a reminder/placeholder

Reply:
doesn't look like it to me, it was the initial implementation.

Reply:
https://github.com/MHBauer/service-catalog/commit/5abaa81844d051891e9170a613b48b6078a50574#diff-c58f3f72bc2cfd04f51929bb4f19ce60R37

Reply:
Can add them all with omitempty if you want?
https://github.com/openservicebrokerapi/servicebroker/blob/v2.12/spec.md#body-6